pyriksdagen/metadata.py

```diff
@@ -347,7 +392,10 @@ def load_Corpus_metadata(metadata_folder=None):
 
 	# Remove redundancy and split file
 	corpus = corpus.drop_duplicates()
+	print(len(corpus))
```

Comment:
Are these necessary?

Reply:
no, should have removed them

Reply:
will remove

Reply:
[2f568b0](https://github.com/swerik-project/pyriksdagen/pull/1/commits/2f568b0f31086ef8ba88a8b6677070bcfc10c12e)